proto/v1/vulnerability.proto

```diff
@@ -73,8 +73,16 @@ message VulnerabilityNote {
     // Required. The package this vulnerability affects.
     string affected_package = 5;
 
-    // Required. The minimum version of the package this vulnerability affects.
-    grafeas.v1.Version min_affected_version = 6;
+    // A vulnerability can affect a package between version numbers that
+    // are disjoint sets of intervals (Example: [1.0.0-1.1.0], [2.4.6-2.4.8] and 
+    // [4.5.6-4.6.8]. 
+    // Required. The starting_affected_version is the starting version number of
+    // one of thoes intervals.
+    grafeas.v1.Version starting_affected_version = 6;
+
+    // Required. The ending version of an interval in which this vulnerability
+    // exists.
+    grafeas.v1.Version ending_affected_version = 11;
 
     // The [CPE URI](https://cpe.mitre.org/specification/) this vulnerability
     // was fixed in. It is possible for this to be different from the
```

Comment:
fixed_cpe_uri:
The distro recommended CPE to update to that contains a fix for this vulnerability.

fixed_package:
The distro recommended package to update to that contains a fix for this vulnerability.

fixed_version:
The distro recommended version to update to that contains a fix for this vulnerability.

Reply:
Done.